docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -12,7 +18,9 @@ Please refer to any Github issues related to the change by the issue number.
  * zonemaster-backend - [Changes](https://github.com/zonemaster/zonemaster-backend/blob/master/Changes)
  * zonemaster-gui - [Changes](https://github.com/zonemaster/zonemaster-gui/blob/master/Changes)
 
-## 2. Set all version numbers
+## 3. Set all version numbers
+
+Update `our $VERSION` for each component to the version number used in the `Changes` file.
```

Comment:
This not true for GUI. See zonemaster/zonemaster-gui#24 for what files to update for GUI.

Reply:
Removed this commit.